### PR TITLE
Support tuples of child geoms directly, wrap single child geom in tuple

### PR DESCRIPTION
**Pull request by asinghvi17**

### CODE REVIEW

**Comment by rafaqz** [comment]
To avoid the allocation?

**Comment by asinghvi17** [comment]
Yeah, just to make it simple.  This way I can use this in GeometryOps on all the geos methods too, and I don't have to call tuples.

**Comment by rafaqz** [comment]
It's slightly concerning for type stability. Now vectors of these geoms have length in the types

File: src/wrappers.jl [hunk 1]
```diff
@@ -180,8 +180,8 @@ for (geomtype, trait, childtype, child_trait, length_check, nesting) in (
         # Is geom a single geometry ?
         if isgeometry(geom)
             if geomtrait(geom) isa $child_trait
-                # If geom is a child_trait, then make geom a vector and call again
-                return $geomtype([geom]; extent, crs)
+                # If geom is a child_trait, then make geom a tuple and call again
+                return $geomtype((geom,); extent, crs)
```
Comment (asinghvi17) [inline src/wrappers.jl:184]: If it's bad for type stability, we can always revert this.  Users can choose to simply write `MultiType((geom,))` if it suits them.

**Review by alex-s-gardner — COMMENTED** [review]
looks good to me

**Comment by alex-s-gardner** [comment]
@rafaqz Is this getting rejected?

**Comment by rafaqz** [comment]
Quite possibly ;)

**Comment by asinghvi17** [comment]
Let's make this a different PR then, I'm happy with the first one

**Comment by rafaqz** [comment]
Probably people can just use Static arrays if they need this? It's more robust as the non mutating vector methods will still work

**Comment by rafaqz** [comment]
Lets close this, I think StaticArrays is better

**Pull request closed without merging**